src/sparcscore/ml/models.py

```diff
@@ -158,6 +161,41 @@ def vgg(cfg, in_channels,  **kwargs):
         model = VGG2(self.make_layers(self.cfgs[cfg], in_channels), **kwargs)
         return model
 
+class VGG2_single_output(VGGBase):
```

Comment:
maybe better name VGG2_regression since you can also make single_output with the other implementation but this one does not have the softmax